code/__HELPERS/unsorted.dm

```diff
@@ -1474,3 +1474,49 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			return "[number]-tuple"
 
 #define TURF_FROM_COORDS_LIST(List) (locate(List[1], List[2], List[3]))
+
+
+/**
+ * One proc for easy spawning of pods in the code to drop off items before whizzling (please don't proc call this in game, it will destroy you)
+ *
+ * Arguments:
+ * * specifications: special mods to the pod, see non var edit specifications for details on what you should fill this with
+ * Non var edit specifications:
+ * * target = where you want the pod to drop
+ * * path = a special specific pod path if you want, this can save you a lot of var edits
+ * * style = style of the pod, defaults to the normal pod
+ * * spawn = spawned path or a list of the paths spawned, what you're sending basically
+ * Returns the pod spawned, in case you want to spawn items yourself and modify them before putting them in.
+ */
+/proc/podspawn_path(specifications)
```

Comment:
Does the name `podspawn_path` make sense anymore with the new signature? Why not just `podspawn`?

Reply:
i wanted to do podspawn_path, but uuh... podspawn is the admin verb! I mean, maybe i'll still do it

Reply:
`podspawn` doesn't look like it's the name of the verb, though.

Reply:
As in, the name of the proc of the verb.